apps/LineRider/install

```diff
@@ -2,40 +2,39 @@
 
 #Bot automatically updates this when I make a new release.
 version=2023.1.1
-#Buster version
-version2=2022.5.1
 
 #Install Depends
 install_packages apt-transport-https dirmngr gnupg ffmpeg mono-devel unzip zip || exit 1
 
 mkdir -p ~/LineRider
 cd ~/LineRider || error "Could not move to folder"
 
-if [[ $(get_codename) == "buster" ]]; then
-  use_version="$version2"
-else
-  use_version="$version"
-fi
-
 #Download LRA-Community-Edition build from repo
 status "Downloading LineRider... "
-wget -O linerider.zip https://github.com/Sussy-OS/LRA-Community-Edition/releases/download/${use_version}/LineRider.game.auto-release.${use_version}-LR.zip || error "Could not wget LineRider from GitHub!"
+wget -O linerider.zip https://github.com/Sussy-OS/LRA-Community-Edition/releases/download/${version}/LineRider.game.auto-release.${version}-LR.zip || error "Could not wget LineRider from GitHub!"
 
 #Install
-unzip linerider.zip || error "Could not unzip the files"
-rm linerider.zip
+unzip linerider.zip || error "Could not unzip linerider.zip!"
+rm linerider.zip || error "Failed to delete linerider.zip!"
+cd ~/
+sudo mv -f ~/LineRider /opt/LineRider || error "Failed to move source folder to /opt!"
+sudo chmod +x /opt/LineRider/linerider.exe
 
 #Add system ffmpeg to expected location
 mkdir -p ~/Documents/LRA/ffmpeg/linux
 cp $(command -v ffmpeg) ~/Documents/LRA/ffmpeg/linux/ffmpeg
```

Comment:
this will break if ffmpeg ever updates and changes ABI

this should be a symlink

Reply:
So, doing what I did before to make a symlink? The app is fine, just Linux for some reason is not liking directly running the command, it wants it there in the directory.

Reply:
Also- why would it break?

Reply:
You are copying the binary. When (not if) ffmpeg updates the binary you copied will be out of date. ffmpeg dynamically links to libraries such as libavcodec5X. Any changes to ABI (such as the pi team updating their decoder) will cause all of these libraries to be updated. Since your copied file does not get updated, it will do multiple things. It will spew errors when running, it may fail to link, and it may segfault

Reply:
> The app is fine, just Linux for some reason is not liking directly running the command, it wants it there in the directory.

No, it is your app. Look in it's code. Somewhere you are telling it to look for ffmpeg in the home folder in that directory.

Reply:
> Well, the ABI documentation says it's fully backwards compatible..

No. Read again what I said and understand it.
I will not merge until this error is fixed.

Reply:
Okay, finding the code and fixing right now.

Reply:
Found the code, and it seems like it does this for all operating systems: https://github.com/Botspot/pi-apps/pull/2253#discussion_r1091354303 I'm not entirely sure how to make it update, or no longer need ffmpeg in that directory, do you think you could help?

Reply:
Sorry, wrong link, updated just now.

Reply:
Oh, I understand, sorry. Now that I've fixed that, can we mark this review as resolved?

Reply:
Just tested it on both Buster and Bullseye, and LineRider works, and the symlink was made properly.